<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualWindow</class>
 <widget class="QDialog" name="ManualWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>20</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stepUpButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Increase Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stepDownButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Decrease Step</string>
   </property>
  </widget>
  <widget class="QPushButton" name="onOffButton_2">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>440</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>On</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="curTemp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>741</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>100</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="clock">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>290</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12:00 AM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="stepSize">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>480</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
